src/osd/PGBackend.cc

```diff
@@ -866,8 +866,6 @@ map<pg_shard_t, ScrubMap *>::const_iterator
       auth = j;
       *auth_oi = oi;
       auth_version = oi.version;
-      auth_bl.clear();
```

Comment:
@tchaikov The original auth_bl wasn't returned to the caller or used for anything other than determining whether to set "object_info_inconsistency" error.  That is why I renamed it to first_bl and moved it out of the auth/auth_oi handling which are returned to the caller.